packages/wallets/src/features/cfd/components/CFDPlatformsListAccounts/CFDPlatformsListAccounts.tsx

```diff
@@ -45,8 +46,8 @@ const CFDPlatformsListAccounts: React.FC<TProps> = ({ onMT5PlatformListLoaded })
     return (
         <div className='wallets-cfd-list-accounts__content'>
             {/* TODO: Update loader with updated skeleton loader design */}
-            {!isMT5FetchedAfterMount && <TradingAppCardLoader />}
-            {isMT5FetchedAfterMount &&
+            {isMT5Loading && <TradingAppCardLoader />}
+            {!isMT5Loading &&
```

Comment:
Wouldn't using isLoading here cause the previous issue we had where the loader does not wait for the actual accounts to be loaded 


https://github.com/binary-com/deriv-app/assets/142860499/a8d285c6-a54c-4091-9f3e-a7c4d0cf3a64